Gemfile.lock

```diff
@@ -1,131 +0,0 @@
-GEM
```

Comment:
please don't remove these files - we kinda need them to know what we built with so dev and prod match.